site.yml

```diff
@@ -53,7 +55,9 @@
     - role: inspec
       install_inspec_controls: [common]
       tags: [common,inspec]
-      when: inspec.enabled|default("False")|bool
+      when:
+        - inspec.enabled|default("False")|bool
+        - ursula_os == 'rhel'
```

Comment:
I don't think you need this. The RHEL7 STIG checks are only installed if ursula_os == 'rhel' so there is no need to skip it here in site.yml